src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -94,4 +117,22 @@ public void testJoinPackageNamesWhichContainNullWithClassName() throws Exception
             String.valueOf(method.invoke(PackageObjectFactory.class, className, packages));
         assertEquals(className, actual);
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testNameToFullModuleNameMap() throws Exception {
+        final Reflections reflections = new Reflections(new ConfigurationBuilder()
```

Comment:
`CheckUtil.getCheckstyleModules` is fixed in latest master. It should return all modules now.